backend/src/main/java/pl/edu/agh/apdvbackend/models/FieldParser.java

```diff
@@ -0,0 +1,25 @@
+package pl.edu.agh.apdvbackend.models;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "field_parser")
+@Getter
+@Setter
+@NoArgsConstructor
+public class FieldParser {
+    @Id
+    @GeneratedValue
+    @Schema(required = true)
+    private Long id;
+
+    @Schema(required = true)
+    private String path;
```

Comment:
nie kumam po co nam tu string "path"

Reply:
Ten `path` to ścieżka do konkretnego pola w JSON'ie, np. dla temperatury może być `/data/senson/temperature`